unit_tests/log_transfer/test_main_chkpt_info.cpp

```diff
@@ -640,3 +640,54 @@ log_system_tdes::map_all_tdes (const map_func &func)
 {
   assert (false);
 }
+
+LOG_TDES *
+logtb_rv_find_allocate_tran_index (THREAD_ENTRY *thread_p, TRANID trid, const LOG_LSA *log_lsa)
+{
+  assert (false);
```

Comment:
When load/recovery unit test is added, some of these functions will need a mock implementation. E.g. here a new tdes is created if one does not exist already.